versions/development/SPEC.md

```diff
@@ -302,7 +302,13 @@ For more details on the postfix quantifiers, see the section on [Optional Parame
 
 For more information on type and how they are used to construct commands and define outputs of tasks, see the [Data Types & Serialization](#data-types--serialization) section.
 
-#### Custom  Types
+#### Numeric Behaviour
+
+`Int` and `Float` are the numeric types.
+`Int` can be used to hold any integer, there is no size limit.
```

Comment:
None? 

Imagine in the Java world, even using `BigInteger` you're still limited by the maximum size of a `String`.

Reply:
I mean, none of our languages are turing complete for the same reason. `If the size would be unrepresentable by the machine, the program must crash` - but this kind of behaviour is assumed. I guess I am trying to say: authors of wdl, do not worry about that machine. Authors of engine, do not make your implementation overly limited.

Reply:
TBH based on what your'e trying to convey, I'd just either stick with keeping it simple and declaring `Int` to be an integer type or go with something fairly standard and declaring it to be a 64 bit integer (i.e. what most ppl would call a long)